### PR TITLE
Add middleware examples for Django.

### PR DESCRIPTION
**Pull request by carlosalberto**
Notes about the samples:

1. Decided to use small, minimal, self standing samples for each sample (which, in turn, mirrors the samples for libhoney-js). The alternative is to only include the actual middleware file for each, and then include instructions to put it in a live example.

2. Python doesn't have anything, out of the box, like process.memoryUsage(), so I decided to simply use similar data (not memory related, though).

### CODE REVIEW

File: examples/django_dynamic_fields/django_dynamic_fields/wsgi.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "django_response_time.settings")
```
Comment (christineyen) [inline examples/django_dynamic_fields/django_dynamic_fields/wsgi.py]: s/`django_response_time`/`django_dynamic_fields`

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+1. `virtualenv examples_env`
+2. `source examples_env/bin/activate`
+2. `(cd ../ &&  python setup.py install) # this installs libhoney`
+3. `pip install django==1.8`
+
+And then on each example:
+
+4. `cd $example-dir`
+5. `python manage.py migrate # initialize the project`
+6. `HONEY_WRITE_KEY=YOUR_WRITE_KEY HONEY_DATA_SET=YOUR_DATASET python manage.py runserver`
```
Comment (christineyen) [inline examples/README.md]: the env variable is `HONEY_DATASET` everywhere else :)

**Comment by carlosalberto** [comment]
Hey Christine,

Thanks for the update. Updates the typos/README as suggested (hope you don't mind I literally put your own suggested updates for the README file ;) )

Let me know :)

**Comment by christineyen** [comment]
Not at all \o/ Thanks!

**Pull request merged**